### PR TITLE
[Media] Testplan update

### PR DESCRIPTION
**Pull request by xlecours**
This pull request update the testplan to reflect the while-you-type filtering

### CODE REVIEW

File: modules/media/test/TestPlan.md [hunk 1]
```diff
@@ -84,12 +84,11 @@ In order to use the media module the user might need one or both of the followin
...
-    - The page should display a data table with records corresponding to this PSCID only
+  1. Under **Browse** tab, a selection filter should be present on top of the page containing the following fields: PSCID, Visit Label, Instrument, File name, For Site, File type and Uploaded by.
+    - PSCID, File name and Uploaded by are text fields, whereas other fields are dropdowns with options pre-filled based on the current project.
+    - Default option of dropdown should be blank.
+  2. Type text in the PSCID input and verify that the table gets filtered as you type.
+  2. Type text in the File name input and verify that the table gets filtered as you type.
+  2. Type text in the Uploaded by input and verify that the table gets filtered as you type.
   4. Select Instrument, Visit Label and For Site (independently and combined) to filter table further
-    - Clicking 👉 **Show Data** should update the table and display filtered records accordingly
+    - The table should update iand display filtered records accordingly
```
Comment (johnsaigle) [inline modules/media/test/TestPlan.md]: Little typo here!

File: modules/media/test/TestPlan.md [hunk 2]
```diff
@@ -84,12 +84,11 @@ In order to use the media module the user might need one or both of the followin
...
-    - PSCID is a text field, whereas other fields are dropdowns with options pre-filled based on the current project.
-    - Default option is "Any"
-  2. Enter an invalid PSCID and click 👉 **Show Data**
-    - The page should display a message "No results found"
-  3. Enter a valid PSCID and click 👉 **Show Data**
-    - The page should display a data table with records corresponding to this PSCID only
+  1. Under **Browse** tab, a selection filter should be present on top of the page containing the following fields: PSCID, Visit Label, Instrument, File name, For Site, File type and Uploaded by.
+    - PSCID, File name and Uploaded by are text fields, whereas other fields are dropdowns with options pre-filled based on the current project.
+    - Default option of dropdown should be blank.
+  2. Type text in the PSCID input and verify that the table gets filtered as you type.
```
Comment (johnsaigle) [inline modules/media/test/TestPlan.md]: Several points marked '2.'

File: modules/media/test/TestPlan.md (hunk 1 above)
Comment (johnsaigle) [inline modules/media/test/TestPlan.md]: Little typo here!

Comment (xlecours) [inline modules/media/test/TestPlan.md]: thanks

**Pull request merged**